tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
+
+## Game Rules:
+
+As we said, it's a two-player game, where one player takes one symbol(**X**) and the second player takes one symbol(**O**).
+
+The first player to get 3 of her marks in a row (**up, down, across, diagonally**) is the winner.
```

Comment:
his/her marks in a row, column or diagonal is the winner. (we can't call a row as column and vice versa) Be specific with your words.